command/agent/job_endpoint.go

```diff
@@ -1385,6 +1394,12 @@ func ApiCSIPluginConfigToStructsCSIPluginConfig(apiConfig *api.TaskCSIPluginConf
 	return sc
 }
 
+func ApiActionToStructsAction(job *structs.Job, action *api.Action, act *structs.Action) {
+	act.Name = action.Name
+	act.Args = slices.Clone(action.Args)
+	act.Command = action.Command
+}
```

Comment:
`Api*ToStructs*` functions should return a new value instead of modifying an input.